catalyst-gateway/bin/src/service/api/cardano/cip36/response.rs

```diff
@@ -142,10 +142,10 @@ pub(crate) struct Cip36Details {
     pub payment_address: Option<common::types::cardano::cip19_shelley_address::Cip19ShelleyAddress>,
     /// If the payment address is a script, then it can not be payed rewards.
     #[oai(default)]
-    pub is_payable: common::types::cardano::boolean::IsPayable,
+    pub is_payable: common::types::generic::boolean::BooleanFlag,
```

Comment:
Why remove the specific types?

Reply:
They are basically just a boolean flags as `BooleanFlag` so just removed redundant types.
The description for this field is overlay the default description of the `BooleanFlag` type.

Reply:
Good point, missed that. Need to take a look is it possible to make a workaround without providing additional types